fetch.bs

```diff
@@ -2985,9 +2983,10 @@ steps:
 
    <li><p>If <var>dataURLStruct</var> is failure, then return a <a>network error</a>.
 
-   <li><p>Return a <a for=/>response</a> whose <a for=response>header list</a> consist of a single
-   <a for=/>header</a> whose <a for=header>name</a> is `<code>Content-Type</code>` and
-   <a for=header>value</a> is <var>dataURLStruct</var>'s <a for="data: URL struct">MIME type</a>,
+   <li><p>Return a <a for=/>response</a> whose <a for=response>status message</a> is
+   `<code>OK</code>`, whose <a for=response>header list</a> consist of a single <a for=/>header</a>
+   whose <a for=header>name</a> is `<code>Content-Type</code>` and <a for=header>value</a> is
+   <var>dataURLStruct</var>'s <a for="data: URL struct">MIME type</a>,
```

Comment:
I know it isn't something you changed in this PR, but there's a repetition of "whose" here which is inconsistent with the similar lines above.

Again, dl/ul could be useful here, especially when it comes to the header list.